src/_gettsim_tests/test_data/grunds_im_alter/2022-07-01/hh_id_25.yaml

```diff
@@ -122,6 +122,12 @@ inputs:
       - 2006
   assumed: {}
 outputs:
+  bruttokaltmiete_m_bg:
```

Comment:
This test currently fails because `bruttokaltmiete_m_bg` and `heizkosten_m_bg` is wrong. This happens because the adult in this household is married to a child ( `kind=True`) below 18. (I pointed this bug out in #746)

I'm not sure what this test is trying to achieve. I'd vote for deleting it or skipping it until `kind` is endogenous.

Reply:
I would also delete the test. This person shouldn't count as `"dependent child living with parents"`

Reply:
Originally, the test was an adult + his/her kid:

```
git log src/_gettsim_tests/test_data/grunds_im_alter/2022-07-01/hh_id_25.yaml
git checkout c56937dc95808927db129ea4a24691e737019e12 -- src/_gettsim_tests/test_data/grunds_im_alter/2022-07-01/hh_id_25.yaml
```

And then the conversion to the pointer structure happened and we ended up with a couple. 

Just revert to the original test.